packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -168,6 +169,7 @@ export function startInterval() {
 
 export async function startTransformByQ() {
     // Set the default state variables for our store and the UI
+    const transformStartTime = Date.now()
```

Comment:
This is passed to startJob and eventually we use it to calculate the latency of the API call, but here it's set long before we call startJob? I think we should just be using `Date.now()` right before we call the API.

Reply:
Since we are removing metrics based on per API call, I want to capture the actual latency from user submitting the form to transformation started (as part of user journey). We can still find latency to the API call on codewhisperer service side.